### ISSUE TITLE
Error 404 "The hostname of the origin is unresolvable (DNS) or blocked by policy." 

### ISSUE THREAD

**Issue by ivanovskiilija** [andrieslouw/imagesweserv#306]
When attempting to open:

https://images.weserv.nl/?url=https://kajgana.com/sites/default/files/styles/facebook/public/2021-09/the-matrix-resurrections-trailer-reaction-cinemacon.jpg?h=7741cde7&itok=qP3LcFZx
https://images.weserv.nl/?url=https://i1.wp.com/gostivarpress.mk/wp-content/uploads/2020/02/infektivno-bolnica-2.jpg?resize=730%2C465&ssl=1&w=140&h=140&t=square&a=attention&bg=white&output=webp

It says: Error 404 "The hostname of the origin is unresolvable (DNS) or blocked by policy."

**Comment by andrieslouw** [comment]
Duplicate of https://github.com/weserv/images/issues/290

**Comment by andrieslouw** [comment]
Please see https://github.com/weserv/images/issues/290

**Comment by ivanovskiilija** [comment]
Hey, 

We have seen the issue about i1.wp.com, but the first link does not use i*.wp.com. Can you give me an answer about that domain?

**Comment by andrieslouw** [comment]
Please see https://domain.opendns.com/kajgana.com

We block all categories mentioned in https://github.com/weserv/images/issues/285#issuecomment-828176654

**Comment by kleisauke** [comment]
I hope this information helped. Please feel free to re-open if questions remain.
